codegen-sbt/src/sbt-test/compiletime-codegen/simple/project/plugins.sbt

```diff
@@ -0,0 +1,13 @@
+import sbt.librarymanagement.Resolver
+
+resolvers += Resolver.mavenLocal
+
+addSbtPlugin("com.github.sbt"            % "sbt-native-packager" % "1.9.4")
```

Comment:
Can you remove the dependencies that are not strictly necessary for this test? More dependencies make it harder to maintain especially when scala/sbt/etc versions change.